src/app/editor/shared/services/record-fixer.service.ts

```diff
@@ -27,46 +27,81 @@ import { JsonUtilService } from './json-util.service'
 
 @Injectable()
 export class RecordFixerService {
-  
+
   private emptyValueService: EmptyValueService = new EmptyValueService();
   private jsonUtilService: JsonUtilService = new JsonUtilService();
 
+  // TODO: return fixed record!
```

Comment:
Are you going to add some code to return it? Or being passed by reference is ok?